theme/static/css/hydroshare_core.css

```diff
@@ -4370,6 +4365,12 @@ div#fb-file-operations-controls {
 #discover-map-legend{
 	z-index: 0;
 	position: absolute;
-	top: 0px;
-	right:0px;
+	top: 0;
+	right:0;
+}
+
+.ui-dialog {
+	-webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+	box-shadow: 0 5px 15px rgba(0, 0, 0, 0.5);
+	z-index: 1031 !important;
```

Comment:
The bootstrap nav bar uses a z-index value of 1030 so I added 1031 for this modal window. If removing the draggable option from the modal window then this line is not needed.